libs/core/src/lib/utils/directives/off-screen-element/off-screen-element.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+export type ariaLiveAssertionType = 'polite' | 'assertive' | 'off';
+export type ariaLiveRelevantType = 'additions' | 'removals' | 'text' | 'all';
+
+/**
+ * This directives puts any html element far left of the screen, so it is not visible.
+ * It includes aria-live attribute, so screen reader will be able to read any change in that element.
+ */
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: '[fd-off-screen], [fdOffScreen]',
+  exportAs: 'fdOffScreen',
+  styleUrls: ['./off-screen-element.component.scss'],
+  template: `<ng-content></ng-content>`,
+  encapsulation: ViewEncapsulation.None,
+  host: {
+    class: 'fd-off-screen'
+  }
+})
+export class OffScreenElementComponent {
+  /** set value for aria-live attribute. possible values are 'polite', 'assertive' and 'off'. Default is 'polite' */
+  @Input()
+  @HostBinding('attr.aria-live')
+  ariaLive: ariaLiveAssertionType = 'polite'
```

Comment:
that property named with `aria...` and others not, should we make it consistent?

Reply:
fixed